### PR TITLE
chore!: remove support for Telescope builtin file_browser

### LINKED ISSUE #12: Removal of `file_browser` from Telescope builtin
Seems like Telescope file_browser was removed as a builtin a while ago. I've been using a custom mapping in my config to use the correct file_browser, but think it'll be nice to update the plugin so it works out of the box. I'm happy to make PR if you want, but just wanted to make sure you're cool with having https://github.com/nvim-telescope/telescope-file-browser.nvim as an extra dependency now.

Relevant change:

https://github.com/nvim-telescope/telescope.nvim/issues/1470#issuecomment-974147513

### PR DESCRIPTION
**Pull request by axieax**
Closes #12 

- [x] Removed support for Telescope builtin file_browser
- [x] Added section to README for using https://github.com/nvim-telescope/telescope-file-browser.nvim

### CODE REVIEW

**Comment by axieax** [comment]
Hi @jvgrootveld could you please review this? Thanks!

**Comment by jvgrootveld** [comment]
Ty for your contribution. I was a little absent lately due to different things. I just want to let everyone know will check all Open issues and PR's shortly.

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ require('telescope').setup{
         ["<C-s>"] = {
           before_action = function(selection) print("before C-s") end,
           action = function(selection)
-            vim.cmd.edit(selection.path)
+            vim.cmd("edit " .. selection.path)
```
Comment (jvgrootveld) [inline README.md:41]: You changed more than described in your commit or PR. Why did you change the `vim.cmd` commands?

File: lua/telescope/_extensions/zoxide/config.lua [hunk 2]
```diff
@@ -12,33 +12,22 @@ local default_config = {
...
-        builtin.file_browser({ cwd = selection.path })
-      end
-    },
     ["<C-f>"] = {
       keepinsert = true,
       action = function(selection)
         builtin.find_files({ cwd = selection.path })
-      end
-    },
-    ["<C-t>"] = {
```
Comment (jvgrootveld) [inline lua/telescope/_extensions/zoxide/config.lua:36]: Curious why you removed this default mapping as well? It is also not described in the comment or PR message of why.

**Comment by jvgrootveld** [comment]
@axieax I reviewed your PR, and I find it awesome that you created a PR to remove an (now) not-working feature and updated the readme with an alternative if others used it. I'm not sure about all the other changes in this PR which are also not described.

Could you explain these extra changes? To merge this feature asap I would recommend to change the PR so that only picker is removed and the Readme is updated.

I could just change it myself but I want to give you the choice to be added as a contributor 😄

**Review by jvgrootveld — CHANGES_REQUESTED** [review]
@axieax I reviewed your PR, and I find it awesome that you created a PR to remove an (now) not-working feature and updated the readme with an alternative if others used it. I'm not sure about all the other changes in this PR which are also not described.

Could you explain these extra changes? To merge this feature asap I would recommend to change the PR so that only picker is removed and the Readme is updated.

I could just change it myself but I want to give you the choice to be added as a contributor 😄

**Comment by jvgrootveld** [comment]
@axieax I notice you are busy. I will wait for your change till next week. After that I will create a copy of your fork and update the code myself.

I am not sure if you will be added as a contributor if I update it but I still will mention you in the PR.

File: lua/telescope/_extensions/zoxide/config.lua (hunk 2 above)
Comment (jvgrootveld) [reply lua/telescope/_extensions/zoxide/config.lua:36]: Updated it myself :)

File: README.md (hunk 1 above)
Comment (jvgrootveld) [reply README.md]: Updated it myself :)

**Pull request merged**